2025-04-10 10:00:20,753 - ERROR - url: https://live1.yangfeiyue.dpdns.org/speedtest/txt/Tianjin_liantong.txt 爬取失败❌, Error: HTTPSConnectionPool(host='live1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Tianjin_liantong.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a72bf0>: Failed to resolve 'live1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:00:30,767 - ERROR - url: https://live1.yangfeiyue.dpdns.org/speedtest/txt/Zhejiang_dianxin.txt 爬取失败❌, Error: HTTPSConnectionPool(host='live1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Zhejiang_dianxin.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a72200>: Failed to resolve 'live1.yangfeiyue.dpdns.org' ([Errno -3] Temporary failure in name resolution)"))
2025-04-10 10:00:45,794 - ERROR - url: https://live1.yangfeiyue.dpdns.org/speedtest/txt/Beijing_liantong.txt 爬取失败❌, Error: HTTPSConnectionPool(host='live1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Beijing_liantong.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a73f40>: Failed to resolve 'live1.yangfeiyue.dpdns.org' ([Errno -3] Temporary failure in name resolution)"))
2025-04-10 10:00:49,859 - ERROR - url: https://live1.yangfeiyue.dpdns.org/speedtest/txt/Shanghai_dianxin.txt 爬取失败❌, Error: HTTPSConnectionPool(host='live1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Shanghai_dianxin.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a73ca0>: Failed to resolve 'live1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:00:53,575 - ERROR - url: https://live1.yangfeiyue.dpdns.org/speedtest/txt/Guangdong_dianxin.txt 爬取失败❌, Error: HTTPSConnectionPool(host='live1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Guangdong_dianxin.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a72590>: Failed to resolve 'live1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:00:57,196 - ERROR - url: https://live1.yangfeiyue.dpdns.org/speedtest/txt/Henan_liantong.txt 爬取失败❌, Error: HTTPSConnectionPool(host='live1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Henan_liantong.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a736a0>: Failed to resolve 'live1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:01:10,273 - ERROR - url: https://live1.yangfeiyue.dpdns.org/speedtest/txt/Hebei_liantong.txt 爬取失败❌, Error: HTTPSConnectionPool(host='live1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Hebei_liantong.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c38e08e0>: Failed to resolve 'live1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:01:15,285 - ERROR - url: https://cai1.yangfeiyue.dpdns.org/speedtest/txt/Shanxi_liantong.txt 爬取失败❌, Error: HTTPSConnectionPool(host='cai1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Shanxi_liantong.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c38e1150>: Failed to resolve 'cai1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:01:20,310 - ERROR - url: https://cai1.yangfeiyue.dpdns.org/speedtest/txt/Shaanxi_dianxin.txt 爬取失败❌, Error: HTTPSConnectionPool(host='cai1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Shaanxi_dianxin.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a73880>: Failed to resolve 'cai1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:01:30,332 - ERROR - url: https://cai1.yangfeiyue.dpdns.org/speedtest/txt/Liaoning_liantong.txt 爬取失败❌, Error: HTTPSConnectionPool(host='cai1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Liaoning_liantong.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a72170>: Failed to resolve 'cai1.yangfeiyue.dpdns.org' ([Errno -3] Temporary failure in name resolution)"))
2025-04-10 10:01:40,354 - ERROR - url: https://cai1.yangfeiyue.dpdns.org/speedtest/txt/Guangxi_dianxin.txt 爬取失败❌, Error: HTTPSConnectionPool(host='cai1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Guangxi_dianxin.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a73b80>: Failed to resolve 'cai1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:01:50,375 - ERROR - url: https://cai1.yangfeiyue.dpdns.org/speedtest/txt/Fujian_dianxin.txt 爬取失败❌, Error: HTTPSConnectionPool(host='cai1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Fujian_dianxin.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c38e0d90>: Failed to resolve 'cai1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:02:00,397 - ERROR - url: https://cai1.yangfeiyue.dpdns.org/speedtest/txt/Sichuan_dianxin.txt 爬取失败❌, Error: HTTPSConnectionPool(host='cai1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Sichuan_dianxin.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c38e0580>: Failed to resolve 'cai1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:02:05,417 - ERROR - url: https://live1.yangfeiyue.dpdns.org/speedtest/txt/Jiangsu_dianxin.txt 爬取失败❌, Error: HTTPSConnectionPool(host='live1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Jiangsu_dianxin.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c38e1a80>: Failed to resolve 'live1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:02:15,197 - ERROR - url: https://live1.yangfeiyue.dpdns.org/speedtest/txt/Anhui_dianxin.txt 爬取失败❌, Error: HTTPSConnectionPool(host='live1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Anhui_dianxin.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a73ca0>: Failed to resolve 'live1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:02:20,213 - ERROR - url: https://live1.yangfeiyue.dpdns.org/speedtest/txt/Hubei_dianxin.txt 爬取失败❌, Error: HTTPSConnectionPool(host='live1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Hubei_dianxin.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a73a30>: Failed to resolve 'live1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:02:20,227 - ERROR - url: https://live1.yangfeiyue.dpdns.org/speedtest/txt/Chongqing_liantong.txt 爬取失败❌, Error: HTTPSConnectionPool(host='live1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Chongqing_liantong.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a72d10>: Failed to resolve 'live1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:02:20,235 - ERROR - url: https://live1.yangfeiyue.dpdns.org/speedtest/txt/Hunan_dianxin.txt 爬取失败❌, Error: HTTPSConnectionPool(host='live1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Hunan_dianxin.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c38e0250>: Failed to resolve 'live1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:02:20,465 - ERROR - url: https://cai1.yangfeiyue.dpdns.org/speedtest/txt/Guangdian.txt 爬取失败❌, Error: HTTPSConnectionPool(host='cai1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Guangdian.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c38e0c40>: Failed to resolve 'cai1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:02:25,508 - ERROR - url: https://cai1.yangfeiyue.dpdns.org/speedtest/txt/Jiangxi_dianxin.txt 爬取失败❌, Error: HTTPSConnectionPool(host='cai1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/Jiangxi_dianxin.txt (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a73760>: Failed to resolve 'cai1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
2025-04-10 10:02:25,519 - ERROR - url: https://cai1.yangfeiyue.dpdns.org/speedtest/txt/NEW.m3u 爬取失败❌, Error: HTTPSConnectionPool(host='cai1.yangfeiyue.dpdns.org', port=443): Max retries exceeded with url: /speedtest/txt/NEW.m3u (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f24c3a727d0>: Failed to resolve 'cai1.yangfeiyue.dpdns.org' ([Errno -2] Name or service not known)"))
